api/bots_api/bots_test_lib.py

```diff
@@ -28,6 +28,32 @@
 class BotTestCase(TestCase):
     bot_name = ''  # type: str
 
+    def get_bot_message_handler(self):
+        # type: () -> Any
+        # message_handler is of type 'Any', since it can contain any bot's
+        # handler class. Eventually, we want bot's handler classes to
+        # inherit from a common prototype specifying the handle_message
+        # function.
+        bot_module = os.path.join(current_dir, "bots",
```

Comment:
I think it would be better if this variable name is `bot_module_path`, since it contains a path str, and to avoid confuses with other "module" term in python.